Automattic/jetpack
# Blocks: Add a Tock block

This is adding a Tock block based on the [Tock Widget plugin](https://wordpress.org/plugins/tock-widget/) 

### Other information:


## Jetpack product discussion
p1HpG7-n0W-p2

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:
With a test site with beta blocks enabled:

* In the editor, insert a Tock block
* Add a URL slug like `roister` or `etch`
* Click Set URL
* Preview the post and check the button works as expected
